.github/dependabot.yml

```
@@ -0,0 +1,10 @@
+version: 2
+updates:
+- package-ecosystem: nuget
+  directory: "/"
+  schedule:
+    interval: daily
+    time: "09:00"
+  open-pull-requests-limit: 10
+  labels:
+  - dependencies
```

Add github actions ecosystem
```suggestion
  - dependencies
- package-ecosystem: "github-actions"
  directory: "/"
  schedule:
    interval: daily
    time: "09:00"
  open-pull-requests-limit: 10
```